Cargo.toml

```diff
@@ -100,7 +100,7 @@ massa_wallet = { path = "./massa-wallet" }
 # Massa projects dependencies
 massa-proto-rs = {git = "https://github.com/massalabs/massa-proto-rs", "rev" = "b45cccca3fb09aa2bbee08aade080127521e39b8"}
 massa-sc-runtime = {git = "https://github.com/massalabs/massa-sc-runtime", "rev" = "4883fe075afca4dd4b7346ed7dff2d506323f9c7"}
-peernet = { git = "https://github.com/massalabs/PeerNet", "branch" = "main" }
+peernet = { git = "https://github.com/massalabs/PeerNet", "rev" = "1d10f55ea883e685691ff527c2466f733ba3e1b2" }
```

Comment:
target main ?

Reply:
No I don't think it's a great idea because if we push a new version and our IDE sync up with it and it has breaking change it will make the compilation fail. I see the rev kinda like a version number we just don't have version on peernet yet but if we had a 0.1.0 we will not point to main but to this version